Sources/App/Fastlane/SlackCommand+Fastlane.swift

```diff
@@ -88,6 +88,38 @@ extension SlackCommand {
         })
     }
 
+    static func runPipeline(
+        metadata: SlackCommandMetadata,
+        branch: String? = nil,
+        ci: CircleCIService,
+        on container: Container
+    ) throws -> Future<SlackResponse> {
+        let pipeline = String(metadata.textComponents[0])
+        let optionsKeysValues = metadata.textComponents.dropFirst()
+            .compactMap { (component: String.SubSequence) -> (String, CircleCIService.PipelineRequest.Parameter)? in
+                let components = component.split(separator: ":")
+                if components.count == 1 {
+                    return (String(components[0]), .bool(true))
+                } else if components.count == 2 {
+                    return (String(components[0]), .string(String(components[1])))
+                } else {
+                    return nil
+                }
+        }
+        var parameters = Dictionary(uniqueKeysWithValues: optionsKeysValues)
```

Comment:
Nit but that means that if we provide an option twice it will make the bot crash.

One use case that could be tempting could be `/stevenson fastlane ui_test_babylon feature:X feature:Y` and (wrongfully) hoping that it would ruin the two features at once. So this misuage could happen.

the other initialize `(keysAndValues:…) { $1 }` could at least easily prevent a crash

Reply:
addressed